### DISCUSSION TITLE
Roku slapping their nameplate on Wyze products

### DISCUSSION THREAD

**Discussion by skilau** [gtxaspec/wz_mini_hacks#342] [General]
https://arstechnica.com/gadgets/2022/10/rokus-new-lights-doorbells-and-switches-are-tv-centered-and-very-familiar/

Should be interesting to see what Roku does with their cam's firmware, and will block wz_mini_hacks from getting in.

Wonder what their idea of "secure boot" is:
Roku states that its products will offer two-factor authentication, user data encryption, and secure boot and will be certified by the [ioXt Alliance](https://www.ioxtalliance.org/).

**Comment by sideup66** [comment]
Its interesting indeed. The hardware itself is based on a Chinese IP camera that I believe you can even find on aliexpress if you look. That version will simply be a dumb IP camera without any of the smart features of the wyze. All wyze did here was make a custom firmware to slap on them and run their services. It seems roku is piggybacking on this setup by the article. If anything, hopefully this paves for greater integration in the stock experience. I wonder if these updates will come to all wyze cams (kind of hope not)
